cmake/Config.cmake.in

```diff
@@ -0,0 +1,6 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(cpuinfo CONFIG)
```

Comment:
Doesn't look right. 

Please test your changes before submitting hunterization patches:
```
Test your changes. Add temporary release to Hunter system and check that
hunter_add_package(foo) is actually working. Do it at least for one toolchain
locally but of course it will be better if you test all of them remotely.
```
- https://docs.hunter.sh/en/latest/creating-new/patch.html

Reply:
Indeed if you add debug message to the `cmake/Config.cmake.in` file:
```
-- [hunter *** DEBUG *** ...] Install to: /.../hunter/_testing/Hunter/_Base/e613dbd/32b203f/5207aed/Install
-- [hunter] CPUINFO_ROOT: /.../hunter/_testing/Hunter/_Base/e613dbd/32b203f/5207aed/Install (ver.: ...)
-- [hunter *** DEBUG *** ...] Package already installed: cpuinfo
-- [hunter *** DEBUG *** ...] load: /.../hunter/_testing/Hunter/_Base/Download/Hunter/unknown/e613dbd/Unpacked/cmake/projects/cpuinfo/hunter.cmake ... end
...
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
Loading cpuinfoConfig.cmake
[1]    segmentation fault (core dumped)  "cmake" "-H/.../hunter/examples/cpuinfo"
```

Side note: nothing to do with Hunter, you will have segmentation fault with any tool that uses standard CMake `find_package` mechanism.